recipe/meta.yaml

```
@@ -16,8 +16,10 @@ build:
     # d1trimfile option is added by a patch and disabled by unsetting SRC_DIR
     # https://github.com/conda-forge/python-feedstock/blob/e7ea437a819f8f06dcbc3769f6e46dc026070c44/recipe/patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
     - set "SRC_DIR="   # [win]
-    # unvendor xsimd; use version packaged by conda
-    - {{ PYTHON }} -c "import os, shutil; shutil.rmtree(os.path.join('pythran', 'xsimd'))"
+    # Set compilers to clang-cl
+    - set "CC=clang-cl.exe"  # [win]
+    - set "CXX=clang-cl.exe" # [win]
+    - set "LDFLAGS=" # [win]
```

```suggestion
    - set "LDFLAGS=" # [win]
```
Is this intentionally left empty? We wouldn't use `none` in its place here?

---

well, LDFLAGS is used to contain flags to pass to the linker: https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html#index-LDFLAGS
setting to `none` would mean that the string literal "none" would be passed to the linker as a flag, and it'd give an error, because it's not a valid linker flag

I guess that this is because LDFLAGS has flags for msvc, not clang-cl? clang-cl tries to be compatible with msvc, though: https://clang.llvm.org/docs/MSVCCompatibility.html. I'm wondering why we would want to use none of our linker flags anyway.

---

@cbouss I'd be interested to know about the second paragraph if you have the time